src/main/java/seedu/dengue/storage/DengueHotspotStorage.java

```diff
@@ -20,9 +23,10 @@ public interface DengueHotspotStorage {
 
     /**
      * Returns DengueHotspotTracker data as a {@link ReadOnlyDengueHotspotTracker}.
-     *   Returns {@code Optional.empty()} if storage file is not found.
+     * Returns {@code Optional.empty()} if storage file is not found.
+     *
      * @throws DataConversionException if the data in storage is not in the expected format.
-     * @throws IOException if there was any problem when reading from the storage.
+     * @throws IOException             if there was any problem when reading from the storage.
```

Comment:
Maybe remove the whitespace? Can't check if it carries over to JavaDoc coz there are too many other errors in the original AB3 LOL

```suggestion
     * @throws IOException if there was any problem when reading from the storage.
```